src/turnstyl.ts

```diff
@@ -124,7 +120,7 @@ class Turnstyl {
         if (JSON.stringify(producerSchema) !== dbPayload) {
           logger.error({
             message:
-              'The database payload and producer event do not match on schema check' +
+              '❌ The database payload and producer event do not match on schema check' +
```

Comment:
```suggestion
              'The database payload and producer event do not match on schema check' +
```

I think this may throw error with an emoji due to changes within our test platform